upup/pkg/kutil/node_api_adapter.go

```diff
@@ -0,0 +1,183 @@
+/*
```

Comment:
Suggest dropping *api* from filename - it seems redundant.. Move to `node_adapter.go`? Thoughts?